modules/code-deployment/src/test/java/org/apache/ignite/deployment/metastore/DeploymentUnitStoreImplTest.java

```diff
@@ -48,39 +54,46 @@
  */
 @ExtendWith(WorkDirectoryExtension.class)
 public class DeploymentUnitStoreImplTest {
+    private static final String LOCAL_NODE = "localNode";
 
     private final VaultManager vaultManager = new VaultManager(new InMemoryVaultService());
 
+    private final List<UnitNodeStatus> history = new ArrayList<>();
```

Comment:
The usage of ArrayList requires tests to run from a single thread otherwise it is not thread-safe. Could you mention this in the comments or use thread-safe collection here?